test/e2e/tests/settings/about-metamask/ui-validation.spec.ts

```diff
@@ -0,0 +1,122 @@
+import assert from 'assert';
+import { Suite } from 'mocha';
+import FixtureBuilder from '../../../fixture-builder';
+import {
+  defaultGanacheOptions,
+  unlockWallet,
+  withFixtures,
+} from '../../../helpers';
+import { Driver } from '../../../webdriver/driver';
+
+const selectors = {
+  accountOptionsMenu: '[data-testid="account-options-menu-button"]',
+  settingsDiv: { text: 'Settings', tag: 'div' },
+  aboutDiv: { text: 'About', tag: 'div' },
+  metaMaskLabelText: { text: 'MetaMask Version', tag: 'div' },
+  metaMaskVersion: { text: '11.7.3', tag: 'div' },
+  headerText: {
+    text: 'MetaMask is designed and built around the world.',
+    tag: 'div',
+  },
+  headingText: { text: 'About', tag: 'h4' },
+  buttonAddressCopy: '[data-testid="address-copy-button-text"]',
+};
+
+async function switchToAboutView(driver: Driver) {
+  await driver.clickElement(selectors.accountOptionsMenu);
+  await driver.clickElement(selectors.settingsDiv);
+  await driver.clickElement(selectors.aboutDiv);
+}
+
+describe('Setting - About MetaMask : @no-mmi', function (this: Suite) {
+  it('validate the view', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        failOnConsoleError: false,
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver }: { driver: Driver }) => {
+        await unlockWallet(driver);
+
+        // navigate to settings and click on about view
+        await switchToAboutView(driver);
+
+        // Validating the heading text
+        const isHeadingText = await driver.isElementPresent(
```

Comment:
Nit: using `isHeadingPresent` is more clear since we are testing the present of text